API/tests/test_endpoints.py

```diff
@@ -69,8 +71,8 @@ def test_create_review(self):
         response = self.client.post("/review", data=self.reviewData)
         review_id = response.data.decode("utf-8").strip().strip("\"")
         print("Test Create Review", review_id)
+        print(response.data)
         self.assertEqual(response.status_code, 200) 
         
         response = self.client.delete(f"/review/{review_id}")
         print("Test Delete Review", response.data)
-        self.assertEqual(response.status_code, 200)
```

Comment:
Can we put this assertion back?

Reply:
do we also need to delete the newly created spot as well?

Reply:
Oh, nice catch! I didn't even notice. I think to follow the outline of the other test then yes, but it doesn't matter too much